### PR TITLE
 Enable spotbugs

### PR DESCRIPTION
**Pull request by MohamedSabthar**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -16,16 +16,15 @@
...
 }
 
+ext.puppycrawlCheckstyleVersion = project.puppycrawlCheckstyleVersion
 ext.ballerinaLangVersion = project.ballerinaLangVersion
-ext.puppycrawlCheckstyleVersion = "8.18"
-ext.testngVersion = "6.14.3"
-ext.slf4jVersion = "1.7.30"
+ext.slf4jVersion = project.slf4jVersion
 ext.stdlibIoVersion = project.stdlibIoVersion
 ext.stdlibLogVersion = project.stdlibLogVersion
```
Comment (ThisaruGuruge) [inline build.gradle:29]: We can remove all these lines

File: gradle.properties [hunk 2]
```diff
@@ -1,7 +1,14 @@
 org.gradle.caching=true
 group=org.ballerinalang
 version=0.7.4-SNAPSHOT
```
Comment (ThisaruGuruge) [inline gradle.properties:3]: Module version hasn't been updated after `0.7.4` release?

File: build.gradle (hunk 1 above)
Comment (MohamedSabthar) [reply build.gradle:29]: removed

File: gradle.properties (hunk 2 above)
Comment (MohamedSabthar) [reply gradle.properties]: updated

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**